resources/assets/js/fieldtype.js

```diff
@@ -78,6 +78,18 @@ Vue.component('video_embed-fieldtype', {
         description: function() {
             // Return the current description (striping html) for vue preview.
             return this.data.description.replace(/<\/?[^>]+(>|$)/g, "");
+        },
+        duration: function() {
+            // Return the current duration for vue preview.
```

Comment:
This can just return the time in seconds or convert the time in seconds (rather than the ISO 8601 time).